### PR TITLE
Historical HS vs New HS fix TM-2282

### PR DESCRIPTION
**Pull request by scott062**
Easy way to test: 
Note -> Use extreme values for expirations to easily tell what is being overwritten or not

> 1. Offer HS
> 2. Check revoke modal values for Reveal and Expiration
> 3. Check same values for info only modal (all of them should align)

> 4. Revoke HS
> 5. View Info Only modal of revoked hs (Should show historical expiration and reveal) -> reveal might be iffy
> 6. View Re-Offer modal version - all those values should be defaulted to be identical to any OTHER brand new offer modal
> 7. Check against brand new offer to make sure both re-offer and offer are identical modals

### CODE REVIEW

File: src/Components/Handshake/HandshakeBureauButton/HandshakeBureauButton.jsx [hunk 1]
```diff
@@ -54,9 +53,7 @@ const HandshakeBureauButton = props => {
         <EditHandshake
           submitAction={submitAction}
           bidCycle={bidCycle}
-          expiration={hs_date_expiration}
-          offer={hs_date_offered}
-          currentlyOffered={hs_status_code === 'handshake_offered'}
+          handshake={handshake}
```
Comment (scott062) [inline src/Components/Handshake/HandshakeBureauButton/HandshakeBureauButton.jsx:56]: Simplified by passing whole HS object

File: src/Components/Handshake/EditHandshake/EditHandshake.jsx [hunk 2]
```diff
@@ -39,8 +69,7 @@ const EditHandshake = props => {
   };
 
   const readOnly = currentlyOffered || infoOnly;
-  const disabledButton = !currentlyOffered &&
-    (isNil(expirationTime) || isPast(validateExpiration()));
+  const userInputError = isNil(expirationTime) || isPast(validateExpiration());
```
Comment (scott062) [inline src/Components/Handshake/EditHandshake/EditHandshake.jsx:72]: more clearly define what these booleans are checking by separating them out

File: src/Components/Handshake/EditHandshake/EditHandshake.jsx [hunk 3]
```diff
@@ -21,11 +31,31 @@ const EditHandshake = props => {
...
+    //   b) CREATE-MODE => show reveal
+    // Case 2:
+    // We have a reveal date and its in the past
+    //  a) INFO-ONLY && (offer_dt < reveal_dt) => show reveal_dt
+    //  b) INFO-ONLY && (offer_dt > reveal_dt) => show offer_dt
+    //  c) CREATE-MODE => show today
+    // Case 3:
+    // No reveal date
+    //   a) INFO-ONLY => show offer_dt
+    //   b) create-mode => show today
```
Comment (scott062) [inline src/Components/Handshake/EditHandshake/EditHandshake.jsx:47]: To be removed - for your testing benefit

**Review by mjoyce91 — APPROVED** [review]

File: src/Components/Handshake/EditHandshake/EditHandshake.jsx (hunk 3 above)
Comment (elizabeth-jimenez) [reply src/Components/Handshake/EditHandshake/EditHandshake.jsx:47]: where's the flowchart

**Pull request merged**